x-pack/plugins/fleet/public/components/agent_enrollment_flyout/steps/agent_enrollment_confirmation_step.tsx

```diff
@@ -46,13 +46,15 @@ export const AgentEnrollmentConfirmationStep = ({
   agentCount,
   showLoading,
   poll = true,
+  isCsp = true,
```

Comment:
nit: should `isCsp` default to `false` here?

Reply:
woops you are right